src/Benchmarks/Middleware/PlaintextMiddleware.cs

```diff
@@ -37,11 +37,15 @@ public static Task WriteResponse(HttpResponse response)
             var payloadLength = _helloWorldPayload.Length;
             response.StatusCode = 200;
             response.ContentType = "text/plain";
-            response.ContentLength = payloadLength;
+
+            // HACK: Setting the Content-Length header manually avoids the cost of serializing the int to a string.
+            //       This is instead of: httpContext.Response.ContentLength = payloadLength;
+            response.Headers["Content-Length"] = "13";
```

Comment:
This would be a regression in 2.0 with the add ContentLength to IHeaderDictionary changes

"React to IHeaderDictionary ContentLength change" #170
"[Perf] Add ContentLength to IHeaderDictionary" aspnet/HttpAbstractions#757
"Use ContentLength as primary data source" aspnet/KestrelHttpServer#1313